plutus-scb/src/Plutus/SCB/MockApp.hs

```diff
@@ -199,14 +200,17 @@ data TxCounts =
 
 txCounts :: Member (State TestState) effs => Eff effs TxCounts
 txCounts = do
-    chain <- use (nodeState . NodeServer.chainState . Wallet.Emulator.Chain.chainNewestFirst)
+    chain <- blockchainNewestFirst
     pool <- use (nodeState . NodeServer.chainState . Wallet.Emulator.Chain.txPool)
```

Comment:
Hmm...I think it's fine that we've moved `blockchainNewestFirst` to top-level, because we want to reuse it elsewhere. But looking at this, what _is_ weird is that we're exporting the function rather than the lens. `blockchainNewestFirst` should be a lens that we `use`. I'll change that.